plugin/trino-hive/src/main/java/io/trino/plugin/hive/SortingFileWriter.java

```diff
@@ -212,12 +213,13 @@ private void mergeFiles(Iterable<TempFile> files, Consumer<Page> consumer)
             Collection<Iterator<Page>> iterators = new ArrayList<>();
 
             for (TempFile tempFile : files) {
-                Path file = tempFile.getPath();
+                String file = tempFile.getPath();
```

Comment:
Previously `configuration.set(FS_DEFAULT_NAME_KEY, "file:///");` was set.
I'm not sure `trinoFileSystem.newInputFile(file);` will default to local file

cc @raunaqmorarka